Specify one, not multiple required APIs for #358 (#363)
ietf-wg-scitt/draft-ietf-scitt-architecture: draft-ietf-scitt-architecture.md

Transparency Services MUST feature a Verifiable Data Structure.
The Verifiable Data Structure records registered Signed Statements and supports the production of Receipts.

> All Transparency Services MUST expose a minimally conformant API ({{-SCRAPI}} for the Registration of Signed Statements and issuance of Receipts.

Reviewer: ```suggestion
All Transparency Services MUST expose the minimally conformant APIs, defined in ({{-SCRAPI}} for the Registration of Signed Statements and issuance of Receipts.
```

Reviewer 2: +1, this makes the MUST clearer for me

Reviewer 3: I sorta wonder why we are making SCRAPI MTI, but I don't object to these specific changes
